### PR TITLE
Add GetOTP API

### PR DESCRIPTION
**Pull request by thoughtgeek**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by thoughtgeek** [comment]
@pawelborkar Could you please merge this?

File: README.md [hunk 1]
```diff
@@ -205,6 +205,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Auth0](https://auth0.com) | Easy to implement, adaptable authentication and authorization platform | `apiKey` | Yes | Yes |
 | [Micro User Service](https://m3o.com/user) | User management and authentication | `apiKey` | Yes | No |
+| [GetOTP](https://otp.dev/en/docs/) | Implement OTP flow quickly  | `apiKey` | Yes | No |
```
Comment (pawelborkar) [inline README.md:208]: Please arrange it in the alphabetical order

**Review by pawelborkar — CHANGES_REQUESTED** [review]
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

File: README.md (hunk 1 above)
Comment (thoughtgeek) [reply README.md:207]: Done

File: README.md [hunk 2]
```diff
@@ -204,6 +204,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Auth0](https://auth0.com) | Easy to implement, adaptable authentication and authorization platform | `apiKey` | Yes | Yes |
+| [GetOTP](https://otp.dev/en/docs/) | Implement OTP flow quickly  | `apiKey` | Yes | No |
```
Comment (pawelborkar) [inline README.md:207]:
```suggestion
| [GetOTP](https://otp.dev/en/docs/) | Implement OTP flow quickly | `apiKey` | Yes | No |
```

Comment (pawelborkar) [reply README.md:207]: each segment must start and end with exactly 1 space

**Review by pawelborkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (thoughtgeek) [reply README.md]: Done.

**Review by pawelborkar — APPROVED** [review]

**Comment by pawelborkar** [comment]
Finally 😄 

Thanks for the contribution @thoughtgeek

**Pull request merged**